src/components/icons/Icons.stories.tsx

```diff
@@ -32,20 +31,15 @@ import {
   AuditedIcon,
   AvadoGlyphIcon,
   BattleTestedIcon,
-  BedrockGlyphIcon,
```

Comment:
Why are we taking these icons out of the story? is it a cleanup for something?

Reply:
@pettinarip not sure why they were removed. I'll review it as soon as possible!

Reply:
Added back those missing icons. Still don't know why they were removed.